src/main/steps/sequence.ts

```diff
@@ -366,7 +366,12 @@ export const sequence: Step[] = [
   },
   {
     url: CHECK_ANSWERS_URL,
-    getNextStep: data => (data.applicant1HelpWithFeesRefNo ? APPLICATION_SUBMITTED : PAY_YOUR_FEE),
+    getNextStep: data =>
+      data.applicationType === ApplicationType.JOINT_APPLICATION
+        ? SENT_TO_APPLICANT2_FOR_REVIEW
+        : data.applicant1HelpWithFeesRefNo
+        ? APPLICATION_SUBMITTED
```

Comment:
spacing is off here which makes this a bit hard to understand

Reply:
This was how the linter cleaned it up

Reply:
ah the linter.